CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## 1.0.0-beta.9 (Unreleased)
 ### New Features
 
+### New Features
+- Azure Communication Common Library
+ - Introduction of  a new struct `CommunicationIdentifierModel` to repesent a union type that is either a `communicationUser`, `phoneNumber`, or `microsoftTeamsUser`.  Along side this new object we introduced `CommunicationIdentifierSerializer` to serialize and deserialize valid JSON. 
```

Comment:
Are not we moving this type to the Chat package? I am also not sure if we should add this here. The type is only public because autorest.swift can't make it internal at this stage; otherwise this type is for internal usage only. @tjprescott , do you have suggestions on if we should include this in the release notes or not.

Reply:
I would say remove it. Is the intent that both the common and the Chat SDKs will have this model internally or is Chat supposed to leverage the implementation in Common? If it's the second case, the the model must be public.

Reply:
The model is going to live in the Chat package. Would it be better instead of having it under `Azure Communication Common Library` to update it to be under the `Azure Communication Chat Library`? Because this change log still does technically apply @RezaJooyandeh @tjprescott

Reply:
Well, it should be in both--removed from AzureCommunication and added to Chat. It would be a breaking change for AzureCommunication, yes?